Source/Libraries/Vanguard/VanguardConnector.cs

```diff
@@ -76,6 +77,7 @@ public object OnMessageReceived(object sender, NetCoreEventArgs e)
         public object SendSyncedMessage(string message) => netConn.SendSyncedMessage(message);
         public object SendSyncedMessage(string message, object value) => netConn.SendSyncedMessage(message, value);
 
+        [SuppressMessage("Microsoft.Design", "CA1822", Justification = "Method is used outside of RTCV, so changing this is higher cost")]
         public void Kill()
```

Comment:
This can be `static`, but I would need to change the callers and it doesn't seem worth it right now. Also, it seems odd that we're calling a no-op function in the clients. Maybe there's context here that I'm missing.

Reply:
This actually appears to be a bug. This should be calling `netconn?.Kill();`   
The current restart on Netcore is very aggressive (because when you're communicating with an unreliable destination, we found it easier to break the rules of graceful shutdown and just go nuclear). In all likelihood, there's another call that's achieving the same goal of killing netcore that's also called.